app/design/adminhtml/Magento/backend/Magento_Sales/web/css/source/module/order/_order-account.less

```diff
@@ -27,3 +27,19 @@
         width: 50%;
     }
 }
+
+.page-create-order {
+    .order-details {
+        &:not(.order-details-existing-customer) {
+            .order-account-information {
+                .field-email {
+                    margin-left: -30px;
+                }
+
+                .field-group_id {
```

Comment:
Hi @amol2jcommerce 
Thank you for contribution and collaboration!

I am not able to find the selector `field-group_id`.
Could you please advice why we need these changes?

Reply:
Hi @sdzhepa, If customer assign any group that time we need these changes.